app/js/components/manageApps/Addon.jsx

```diff
@@ -80,6 +87,61 @@ class Addon extends Component {
 
   }
 
+  fetchCurrentUser(){
+    const apiHelper = new ApiHelper(null);
+    const getUserData = new Promise(function(resolve, reject) {
+      apiHelper.get('/v1/session').then(response => {
+        resolve(response);
+      });
+    });
+    return getUserData;
+  }
+
+  checkIsAdmin() {
+    this.fetchCurrentUser().then((response) => {
+      const user = response.user;
+      if (user) {
```

Comment:
Make the assignment, after the check. In other words put the assignment in the `if` statement

Reply:
done  👍